coralproject/talk
# Moderate bio or username

## What does this PR do?

Adds the ability to moderate a bio or a username to the admin panel.

## How do I test this PR?

1. Flag a users's bio in the comment stream.
2. Check the 'pending' moderation queue, you should see the flagged bio in the stream.
3. Click 'X', you should see a modal warning you about suspending the user.
4. Click 'Next', you should be able to edit an e-mail to the user.
5. Click 'Send', the user's status should be set to "suspended" and the user should be sent an e-mail notification.
